<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTIR Commander</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 3, 48);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 1,23 , 255), stop:1 rgba(0, 1, 71, 255));
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 190, 37);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Sample Name</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="omnicConnected_label">
               <property name="text">
                <string>Omnic Not Connected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tempControllerConnected_label">
               <property name="text">
                <string>Temperature Controller Not Connected</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="sampleName_lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="temp_checkBox">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Lower Temperature (K)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lowerTemp_lineEdit">
               <property name="text">
                <string>290</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Upper Temperature (K)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="upperTemp_lineEdit">
               <property name="text">
                <string>300</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Temperature Step (K)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="stepTemp_lineEdit">
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="voltage_checkBox">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Lower Voltage (V)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lowerVoltage_lineEdit">
               <property name="text">
                <string>-0.1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Upper Voltage (V)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="upperVoltage_lineEdit">
               <property name="text">
                <string>0.1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Voltage Step (V)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="stepVoltage_lineEdit">
               <property name="text">
                <string>0.01</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="user_lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="run_pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Run Sweep</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="Graph" name="temperature_graph" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 1, 71, 0);
border-right-color: rgb(255, 0, 4);
border-top-color: rgb(255, 0, 0);
gridline-color: rgb(239, 255, 89);
border-color: rgb(0, 1, 71);</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QWidget</extends>
   <header>FTIR_Commander.Graph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
